tasks/build/docker/docs/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:16.04
+
+RUN set -x && \
+  apt-get update && \
+  apt-get install -y \
+    git \
+    xsltproc \
+    curl \
+    python \
+    libxml2-utils
+
+RUN groupadd --gid 1010 kibana \
+  && useradd --uid 1010 --gid kibana --shell /bin/bash --create-home kibana
+
+COPY docs /home/kibana/ascii_docs
+RUN chown -R kibana:kibana /home/kibana/ascii_docs
+
+USER kibana
+
+WORKDIR /home/kibana
+RUN git clone --depth 1 https://github.com/elastic/docs.git docs_builder
+
+WORKDIR /home/kibana/docs_builder
+CMD ./build_docs.pl --doc /home/kibana/ascii_docs/index.asciidoc --out /home/kibana/html_docs
```

Comment:
Our docs are built with `--chunk=1`, so you might as well add that to this task as well.  It should only really affect output and shouldn't make a difference as far as successful builds are concerned, but you never know!